packages/itwinui-react/src/core/Stepper/Stepper.tsx

```diff
@@ -21,6 +21,10 @@ export type StepProperties = {
    * A tooltip giving detailed description to this step.
    */
   description?: string;
+  /**
+   * Custom content passed for completed steps.
```

Comment:
I have made the change accordingly. Since we now allow passing custom content in any step (with `isPast` removed), should I add more logic in the docs and stories to indicate which step is completed for the checkmark icon example as in what I have done in the latest [commit](fa6ded83a48e358cd85daafa6b5977e9922d2447), to make it more reasonable for the use of checkmark?
![image](https://github.com/user-attachments/assets/0e8c0499-c6fc-4289-83f1-944b7ae6cc70)

Reply:
I have added `Set` in the doc example.

Reply:
I have replaced the `Set` and used `currentStep` instead and also added the after-PR todo item.